bottlerocket/agents/src/sonobuoy.rs

```diff
@@ -77,6 +77,7 @@ where
         .args(k8s_image_arg)
         .args(e2e_repo_arg)
         .args(sonobuoy_image_arg)
+        .arg("--plugin-env=e2e.E2E_EXTRA_ARGS=--non-blocking-taints=sonobuoy")
```

Comment:
Interesting. What does this do?

Reply:
For normal sonobuoy testing, nothing. For karpenter testing it allows nodes in the cluster to have the NoSchedule taint effect. The taint prevents sonobuoy nodes from being scheduled on the AL tainted nodegroup nodes.